src/components/TheNavBar.vue

```diff
@@ -4,6 +4,13 @@
     class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top"
     id="sideNav"
   >
+    <div id="login" class="navlink">
```

Comment:
I like the way it looks but can we make the "log in" disappear if you are currently logged in?
![image](https://user-images.githubusercontent.com/11019755/184722983-c0cd515b-2076-44cf-974b-d25cb0e45670.png)